test/tests/Sensor/CanSensorAccessoriesTest.cpp

```diff
@@ -1,13 +1,208 @@
+	CanSensorAccessories accessories(interface, CAN_ACC_STATUS);
+
+	Handler::instance().begin();
+
+	for (const auto& pair : statusIdToGetter) {
+		bool statusIsValid = false;
+		REQUIRE ( (accessories.*pair.second)(statusIsValid) == CanSensorAccessories::Unknown );
+		REQUIRE_FALSE ( statusIsValid );
+	}
+}
+
+TEST_CASE( "CanSensorAccessories::update -- test update individual properties", "[CanSensorAccessories][Sensor]" ) {
+	CanBusMock canBusMock(CAN_MESSAGE_AVAIL_TEST);
+	CanInterface interface(&canBusMock);
+	CanSensorAccessories accessories(interface, CAN_ACC_STATUS);
+	setMillis(DEFAULT_START_TIME_MILLIS);
+
+	CanMessage msg = CAN_MESSAGE_NULL;
+	msg.id = CAN_ACC_STATUS;
+	msg.dataLength = 1;
+
+	Handler::instance().begin();
+
+	SECTION("Test update headlights status") {
+		msg.data[0] = (CanSensorAccessories::StatusIdHeadlights << 1) | 0x1;
+		canBusMock.setCanMessage(msg);
+		
+		Handler::instance().handle();
+
+		bool statusIsValid = false;
+		REQUIRE( accessories.getStatusHeadlights(statusIsValid) );
+		REQUIRE( statusIsValid );
+
+		testValidation(accessories, CanSensorAccessories::StatusIdHeadlights);
+	}
+
+	SECTION("Test update brakelights status") {
+		msg.data[0] = (CanSensorAccessories::StatusIdBrakelights << 1) | 0x1;
+		canBusMock.setCanMessage(msg);
+
+		Handler::instance().handle();
+
+		bool statusIsValid = false;
+		REQUIRE( accessories.getStatusBrakelights(statusIsValid) );
+		REQUIRE( statusIsValid );
+
+		testValidation(accessories, CanSensorAccessories::StatusIdBrakelights);
+	}
+
+	SECTION("Test update horn status") {
+		msg.data[0] = (CanSensorAccessories::StatusIdHorn << 1) | 0x1;
+		canBusMock.setCanMessage(msg);
+
+		Handler::instance().handle();
+
+		bool statusIsValid = false;
+		REQUIRE( accessories.getStatusHorn(statusIsValid) );
+		REQUIRE( statusIsValid );
+
+		testValidation(accessories, CanSensorAccessories::StatusIdHorn);
+	}
+
+	SECTION("Test update hazards status") {
+		msg.data[0] = (CanSensorAccessories::StatusIdHazards << 1) | 0x1;
+		canBusMock.setCanMessage(msg);
+
+		Handler::instance().handle();
+
+		bool statusIsValid = false;
+		REQUIRE( accessories.getStatusHazards(statusIsValid) );
+		REQUIRE( statusIsValid );
+
+		testValidation(accessories, CanSensorAccessories::StatusIdHazards);
+	}
+
+	SECTION("Test update right signal status") {
+		msg.data[0] = (CanSensorAccessories::StatusIdRightSignal << 1) | 0x1;
+		canBusMock.setCanMessage(msg);
+
+		Handler::instance().handle();
+
+		bool statusIsValid = false;
+		REQUIRE( accessories.getStatusRightSignal(statusIsValid) );
+		REQUIRE( statusIsValid );
+
+		testValidation(accessories, CanSensorAccessories::StatusIdRightSignal);
+	}
+
+	SECTION("Test update left signal status") {
+		msg.data[0] = (CanSensorAccessories::StatusIdLeftSignal << 1) | 0x1;
+		canBusMock.setCanMessage(msg);
+
+		Handler::instance().handle();
+
+		bool statusIsValid = false;
+		REQUIRE( accessories.getStatusLeftSignal(statusIsValid) == CanSensorAccessories::On );
```

Comment:
Why is this the only section with a `== CanSensorAccessories::On` in the `REQUIRE` statement?

Reply:
I meant to go through and replace all those.  Thanks for catching that